src/app/containers/PageViewAnalytics/labelHelpers/index.test.js

```diff
@@ -0,0 +1,340 @@
+import Cookie from 'js-cookie';
+import onClient from '../../../helpers/onClient';
+import {
+  setWindowValue,
+  resetWindowValue,
+} from '../../../helpers/tests/setWindowValue';
+
+let isOnClient = true;
+
+jest.mock('../../../helpers/onClient', () => jest.fn());
+onClient.mockImplementation(() => isOnClient);
+
+const {
+  getDestination,
+  getScreenInfo,
+  getBrowserViewPort,
+  getCurrentTime,
+  getDeviceLanguage,
+  getAppType,
+  getHref,
+  getReferrer,
+} = require('./index');
+
+let locServeCookieValue;
+
+const returnsNullWhenOffClient = func => {
+  describe('returns null when not on client', () => {
+    beforeEach(() => {
+      isOnClient = false;
+    });
+
+    afterEach(() => {
+      isOnClient = true;
+    });
+
+    it('should find value in good data', () => {
+      expect(func()).toEqual(null);
+    });
+  });
+};
+
+describe('getDestination', () => {
+  const stuff = [
```

Comment:
```suggestion
  const getDestinationTestScenarios = [
```
maybe a name like this would be more suitable?